src/sentry/models/project.py

```diff
@@ -435,3 +436,17 @@ def delete(self, **kwargs):
 
 
 pre_delete.connect(delete_pending_deletion_option, sender=Project, weak=False)
+post_save.connect(
+    lambda instance, **kwargs: update_code_owners_schema.apply_async(
+        kwargs={"organization": None, "integration": None, "projects": [instance.project]}
```

Comment:
we can get the `organization` from the `instance.project.organization`. I think we should plan on always passing in a organization to the task